filters/austerity.go

```diff
@@ -1,15 +1,22 @@
-package main
+package filters
 
 import (
 	"fmt"
 	"math"
 	"math/rand"
+	"sync"
 
-	"github.com/stripe/unilog"
 	"github.com/stripe/unilog/clevels"
 )
 
-func austerityFilter(line string) string {
+var startSystemAusterityLevel sync.Once
+
+func AusterityFilter(line string) string {
+	// Start austerity level loop sender in goroutine just once
+	startSystemAusterityLevel.Do(func() {
```

Comment:
After the first call, this will reduce to 

```go
if atomic.LoadUint32(&o.done) == 1 {
		return
}
```

which should be fine from a performance perspective.

Reply:
Yup, should be ok.